projects/storefrontapp-e2e-cypress/cypress/helpers/my-company/models/MyCompanyConfig.ts

```diff
@@ -0,0 +1,48 @@
+import { MyCompanyRowConfig } from './MyCompanyRowConfig';
+
+export interface MyCompanyConfig {
```

Comment:
Question: For extensibility reasons, maybe all config elements should be optional?